ui/page/governance/proposals_page.go

```diff
@@ -321,11 +324,12 @@ func (pg *ProposalsPage) Layout(gtx C) D {
 							return pg.categoryDropDown.Layout(gtx, pg.orderDropDown.Width+41, true)
 						}),
 					)
-				}),
-			)
-		})
+				})
+			}),
+		)
+		// })
 	}
-	return components.UniformPadding(gtx, pg.splashScreenLayout)
```

Comment:
Why was the splash screen removed?

Reply:
Is this the enable governance screen? If so, Raedah has requested we work towards removing the welcome screens to a more unified layout with on-off slider buttons for each core functionality that is not turned on by default. There are some mockups that fegor made of this. However, this was supposed to happen after 1.0 relase.

Reply:
that screen is now on the governance_page.go and no longer on the proposals_page.go, as the proposals and consensus pages come under governance page